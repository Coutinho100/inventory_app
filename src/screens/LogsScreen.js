import React from 'react';
import { View, Text } from 'react-native';



const LogsScreen = props => {

    return (
        <View>
            <Text>
                LogsScreen
            </Text>
        </View>
    );
};




export default LogsScreen;