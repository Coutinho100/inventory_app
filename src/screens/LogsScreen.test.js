import React from 'react';
import LogsScreen  from './LogsScreen';

import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
import test_function, {findByTestAttr, mockPlatform, findByTestProps, mockAlert} from '../_test_/componenets/test_function';
// import Alert from 'Alert';

// import {act} from 'react-dom/test-utils';
// JestHook.mock('expo-font');

 
configure({
    adapter: new EnzymeAdapter
})
const function_click=jest.fn()



const dummy_data = {

    navigation:{toggleDrawer:function_click, navigate:function_click}

}


describe('<LogsScreen />', () => {

  
    
  it("Events and props test in LogsScreen", () =>
        {
            const wrapper = shallow(<LogsScreen ></LogsScreen>)
            expect(wrapper).toMatchSnapshot();
           const navOption= LogsScreen.navigationOptions(dummy_data)
           expect(typeof navOption).toBe("object");
        });

  
        });


