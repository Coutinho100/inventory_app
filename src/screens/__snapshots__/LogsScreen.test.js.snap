// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogsScreen /> Events and props test in LogsScreen 1`] = `ShallowWrapper {}`;
